gothemis/cell/cell.go

```diff
@@ -191,7 +191,7 @@ var (
 	ErrMissingContext    = errors.NewWithCode(errors.InvalidParameter, "associated context is required in Context Imprint mode")
 	ErrOutOfMemory       = errors.NewWithCode(errors.NoMemory, "Secure Cell cannot allocate enough memory")
 	// Deprecated: Since 0.14. Use ErrOutOfMemory instead.
-	ErrOverflow          = ErrOutOfMemory
```

Comment:
Looks like broken formatting 😢

Reply:
Uh... Well, that's what my `gofmt` insists on being The Go Way™.

As it has nothing to do with this patchset, I'll revert this one too.